typescript/infra/helloworld.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What is this file?

Reply:
Are these contract addresses? Should we write them to `infra/config/environments/testnet2`?